api/v1alpha1/capacity_types.go

```diff
@@ -236,6 +236,22 @@ type Quota struct {
 
 // CapacityStatus defines the observed state of Capacity
 type CapacityStatus struct {
+	// usedResources are the resources (by flavor) currently in use by the
+	// workloads assigned to this capacity
+	UsedResources map[corev1.ResourceName]map[string]Usage `json:"usedResources,omitempty"`
+
+	// runningWorkloads is the number of workloads currently assigned to this
+	// capacity.
+	RunningWorkloads int32 `json:"runningWorkloads,omitempty"`
```

Comment:
Running is a little confuse. Actually the workload is just assigned.

RunningWorkloads --> AssignedWorkloads